ci-operator/config/terraform-redhat/terraform-provider-ocm/terraform-redhat-terraform-provider-ocm-main.yaml

```diff
@@ -91,6 +91,7 @@ tests:
       POLARION_STEP_RESULTS: artifacts/e2e-periodic/openshift-extended-test/artifacts/junit
       TEST_FILTERS: ~MicroShiftOnly&;~HyperShiftMGMT&;~CPaasrunOnly&;~NonPreRelease&;~VMonly&;~ProdrunOnly&;~StagerunOnly&;~DisconnectedOnly&;~Disruptive&;~Serial&;ROSA&
       TEST_SCENARIOS: OLM|Security_and_Compliance|Operator_SDK|Network_Edge|Workloads|Image_Registry|Container_Engine_Tools|MCO|ETCD|PSAP|SDN|Cluster_Infrastructure|API_Server|Authentication|Cluster_Observability|LOGGING
+      TF_FOLDER: ci/e2e/oidc_configuration/cluster_with_managed_oidc_config
```

Comment:
I think we should rename the job name so it will be explicit and clear what flow it does. e.g.:
```
- as: managed-oidc-config-e2e-periodic
```